xarray/tests/test_formatting.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 from textwrap import dedent
 
+from textwrap import dedent
```

Comment:
It looks like changing from another PR have leaked in here? Let's try to figure that out...

Reply:
Seems to have accidentally crept in when I merged changes from pydata/master into my local repo 4~5 days back. Here's what I managed to trace (from latest to earliest instance of test_formatting.py being changed):
https://github.com/Ouranosinc/xarray/pull/14
https://github.com/jwenfai/xarray/commit/31ccebf984c07d2fe337e5a5f6f589000346cade
https://github.com/jwenfai/xarray/commit/9fbb0162bf30374a712e6b74cac9e73fa544ae20

Reply:
It looks like it’s just a bad merge — these tests are now duplicated twice. You can simply delete the redundant code and push a new commit.